source/RoslynAnalyzers/MessageContractAnalyzers.cs

```diff
@@ -56,17 +58,22 @@ void CheckNode(SyntaxNodeAnalysisContext context)
             // first we have to figure out which kind of type we've encountered (if any)
             var apiSurfaceType = DetermineApiSurfaceType(typeDec);
             if (apiSurfaceType == null) return;
+            
+            var attrNames = typeDec.AttributeLists.SelectMany(al => al.Attributes.Select(a => (a.Name as IdentifierNameSyntax)?.Identifier.Text));
+            // some rules are applied differently to types marked experimental (e.g. BFF's)
```

Comment:
Tangent: We should put a new attribute into server to mark these as "not intended as part of the public api", rather than "experimental"... It's not quite the right word.

Reply:
Agree, that'd be nice